http-receive-edi-850-855/readme.md

```diff
@@ -0,0 +1,111 @@
+# Expose B2B Cloud for HTTP/HTTPS partners to exchange EDI messages
+
+B2B Cloud can be exposed as HTTP/HTTPS channel and used to exchange EDI messages with Partners. This usecase is common for all Partners who has capability of exchanging EDI messages over HTTP which is a very popular protocol for EDI.
+
+Contributors: Chaitanya Jadcherla, Mangat Rai , Shashank Patel
+
+
+## Prerequisites
+1. You need Software AG webmethods.io B2B cloud tenant and webmethods.io integration cloud tenant. If you don't have one; sign up for free 30 trial tenant at [Software AG B2B](https://signup.softwareag.cloud/#/?product=b2b)
+
+![](https://github.com/mangatrai/webmethods-b2b-examples/blob/master/http-receive-edi-850-855/images/B2BLandingPage.png)
```

Comment:
Lets make these relative! For example:
Change:
```
![](https://github.com/mangatrai/webmethods-b2b-examples/blob/master/http-receive-edi-850-855/images/B2BLandingPage.png)
```
To:
```
images/B2BLandingPage.png
```
and it will work locally too.

Reply:
Made changes as per your suggestion